Common/src/CConfig.cpp

```diff
@@ -1161,6 +1161,8 @@ void CConfig::SetConfig_Options() {
   addDoubleOption("GAS_CONSTANT", Gas_Constant, 287.058);
   /*!\brief GAMMA_VALUE  \n DESCRIPTION: Ratio of specific heats (1.4 (air), only for compressible flows) \ingroup Config*/
   addDoubleOption("GAMMA_VALUE", Gamma, 1.4);
+  /*!\brief THERMODYNAMIC_PRESSURE  \n DESCRIPTION: Thermodynamics(operating) Pressure (101325 Pa), only for incompressible flows) \ingroup Config*/
+  addDoubleOption("THERMODYNAMIC_PRESSURE", Pressure_Thermodynamic, 101325.0);
```

Comment:
Sounds like having the right value is very important? Does it make sense to make this not optional?

Reply:
Hi Pedro, Yes, it is important because currently the thermodynamic pressure is computed in the CIncEulerSolver as

$p_{inc}=\rho_{inc}*R*T_{inc}$

where Inc means the INC_DENSITY_INIT and INC_TEMPERATUR_INIT and the R is the gas constant defined as $R_{u}/M$ , where $R_{u}$ is the universal gas constant and $M$ molecular weight of the first species (first gas). For a mixture of 2 gases, the $\rho_{inc}, T_{inc}, M$ can be set in the cfg file in order to obtain the desired operating pressure (usually 101325 pa for combustion) , but when the mixture is more than 2 gases, setting $\rho_{inc}, T_{inc}, R$ is more difficult because they depend on the composition at the inlets. So we could have two possibilities: 1) before setting the operating pressure , we could do two loops for computing the density and M in order to obtain the operating pressure or 2) set the operating pressure as a user input and  avoid these two additional loops, which is the way that we think is better and also allows using be default the use of the operating pressure which is usually considering in combustion problems (101325 pa) if the operating pressure is not provided in the cfg file. Please let me know f you have some suggestions/ideas about how this could be improved. Thank you so much in advance :) .

Reply:
Understood, thank you for the explanation.
Then I would say it is important to document and make sure this is included in the examples.
I leave it up to you and @bigfooted to decide if you want to enforce this to have been set by the user.

Reply:
Thanks for the suggestions. We are working in examples, we plan soon to upload a tutorial explaining the composition-dependent model. Thank you so much!!

Reply:
Hi Pedro, yesterday we decided with @bigfooted to enforce the thermodynamic pressure to be set by the user as it is in this pull request. Please let me know if there is something else to be done for this pull request. Thank you so much in advance!!

Reply:
Hi Pedro, thanks for your comment. Yes, we want to enforce it, but we also want that if the thermodynamic pressure is not provided in the .cfg file, the default value will be used. Similar to what happens with gamma, when it is not provided, SU2 considers the default value as 1.4.

Reply:
That's not enforcing anything, that's doing nothing...
Update the config_template and put this options explicitly in the testcases that now use it, and document what it does there.

Reply:
Understood. I will do it. Thanks for your comment!!